Python/optimizer_bytecodes.c

```diff
@@ -708,6 +708,16 @@ dummy_func(void) {
         sym_set_type(callable, &PyFunction_Type);
     }
 
+    op(_CHECK_METHOD_VERSION, (func_version/2, callable, null, unused[oparg] -- callable, null, unused[oparg])) {
+        if (sym_is_const(ctx, callable) && sym_matches_type(callable, &PyMethod_Type)) {
+            assert(PyMethod_Check(sym_get_const(ctx, callable)));
+            PyMethodObject *method = (PyMethodObject *)sym_get_const(ctx, callable);
```

Comment:
It's more efficient to do it like this:

```suggestion
            PyMethodObject *method = (PyMethodObject *)sym_get_const(ctx, callable);
            assert(PyMethod_Check(method)));
```